cmd/generate.go

```diff
@@ -80,11 +80,10 @@ func newGenerateCmd() *cobra.Command {
 				},
 
 				OSImageConfig: map[datamodel.Distro]datamodel.AzureOSImageConfig{
-					datamodel.Ubuntu:         datamodel.Ubuntu1604OSImageConfig,
-					datamodel.Ubuntu1804:     datamodel.Ubuntu1804OSImageConfig,
-					datamodel.Ubuntu1804Gen2: datamodel.Ubuntu1804Gen2OSImageConfig,
```

Comment:
we need this in AKS. But i'm not sure if we need the whole osImageConfig for agentbaker. So if you keep osImageConfig, please keep Ubuntu1804Gen2

Reply:
I removed the whole OSImageConfig